src/js/workshop-comments.js

```diff
@@ -0,0 +1,144 @@
+const commentForm = document.getElementById("comment-form");
+
+const nameDOMElement = document.getElementById("name");
+const commentDOMElement = document.getElementById("comment");
+
+commentForm.addEventListener("submit", (event) => {
+	event.preventDefault();
+
+	const nameDOMElement = document.getElementById("name");
+	const name = nameDOMElement.value.trim();
+	const commentDOMElement = document.getElementById("comment");
+	const comment = commentDOMElement.value.trim();
+
+	const submittedCommentMessage = document.querySelector(".post-failure-message");
```

Comment:
It might be better to rename the variable "submittedCommentMessage" to "submitFailureMessage".

Reply:
Please rename the DOM selector for the failure message to `.submit-failure-message` for consistence.

Please Rename the corresponding variable and DOM selector for the success message in the way. Thanks.